### DISCUSSION TITLE
Can't see my actual ticket on github ?!

### DISCUSSION THREAD

**Discussion by mouadessalim** [community/community#34896] [Other Feature Feedback, Questions, & Ideas]
Hey i've recetly submitted a ticket to github and recevied a confirmation mail but i can't see it on the github support page i am actually connected to the correct account. Is it normal or can someone help me ?

**Comment by antonblr** [comment]
So you see nothing here - https://support.github.com/tickets/personal/0 ? But regardless, you should be able to communicate by replying to that email.

**Reply by mouadessalim** [reply]
No actually i tried to create another ticket in another section and i can see it. When i compared the mails received there is some difference look below : 

![image](https://user-images.githubusercontent.com/38190847/193478946-b0ccbc1c-38b7-48d6-9b52-b6c7f63165e7.png)
![image](https://user-images.githubusercontent.com/38190847/193478971-5d32e712-80fc-4c35-a860-d6998a2649f6.png)

you can see that in one mail there is `view or update` and in the other there is just `Ticket ID: 1817728`.

**Comment by Kizyki143** [comment]
[1817728](tel:1817728)
